cmd/kubeadm/app/apis/kubeadm/v1alpha2/defaults.go

```diff
@@ -59,7 +59,7 @@ const (
 	// DefaultProxyBindAddressv6 is the default bind address when the advertise address is v6
 	DefaultProxyBindAddressv6 = "::"
 	// KubeproxyKubeConfigFileName defines the file name for the kube-proxy's KubeConfig file
-	KubeproxyKubeConfigFileName = "/var/lib/kube-proxy/kubeconfig.conf"
+	KubeproxyKubeConfigFileName = "/var/lib/kubelet/pods/"
```

Comment:
@xichengliudui 
thanks for the PR, but i don't think this is valid. :)

this constant should hold a path to the user provided kubeconfig file for kube-proxy (e.g. `kubeproxy --kubeconfig`)and not to the folder that has the kubelet pod data.

Reply:
@xichengliudui please, checkout the comment [here](https://github.com/kubernetes/kubernetes/blob/426ef9d349bb3a277c3e8826fb772b6bdb008382/staging/src/k8s.io/apimachinery/pkg/apis/config/types.go#L21).